service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -73,7 +74,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         return respond(caseDetails);
     }
 
-    private void run872(CaseDetails caseDetails) {
+    private void run1144(CaseDetails caseDetails) {
         CaseData caseData = getCaseData(caseDetails);
```

Comment:
On second thoughts - does this line not throw an exception due to the deserialisation step? (as here it's still EDIT_HEARING)